### ISSUE TITLE
umd: true does not seem to work on the UMD definition used in bluebird.

### ISSUE THREAD

**Issue by qraynaud** [mariusGundersen/amd-optimizer#5]
Maybe the one bluebird is using is not valid?

```js
// …
else if("function"==typeof define&&define.amd)define([],e);
// …
```

**Comment by mariusGundersen** [comment]
I'm open for pull requests for supporting it. Have a look at https://github.com/mariusGundersen/amd-optimizer/blob/master/source/locateUmdModule.js and the unit test for it

**Comment by mariusGundersen** [comment]
Sorry, didn't mean to close it, clicked wrong button!

**Comment by mariusGundersen** [comment]
This has been fixed in version 0.4.2
